EmotiBit.cpp

```diff
@@ -2446,19 +2610,17 @@ void EmotiBit::hibernate() {
 			Serial.println(ul);
 		}
 	}
-
 	// Turn off EmotiBit power
 	Serial.println("Disabling EmotiBit power");
 	pinMode(_hibernatePin, OUTPUT);
-	digitalWrite(_hibernatePin, HIGH);
-	pinMode(_hibernatePin, INPUT);
-
+	digitalWrite(_hibernatePin, emotiBitVersionController.getSystemConstant(SystemConstants::EMOTIBIT_HIBERNATE_LEVEL));
```

Comment:
Power the pin on the assumption that it is V3, (making hibernate pin LOW), seems to stop code exe. for V2. V2 is not able to activate Deep sleep.
Working to fix it.